src/types/builtins.ts

```diff
@@ -0,0 +1,246 @@
+                [new IntType(256, false)],
+                FunctionVisibility.Default,
+                FunctionStateMutability.Pure
+            ),
+            ">=0.4.13"
+        ]
+    ],
+    [
+        "mulmod",
+        [
+            new FunctionType(
+                undefined,
+                [new IntType(256, false), new IntType(256, false), new IntType(256, false)],
+                [new IntType(256, false)],
+                FunctionVisibility.Default,
+                FunctionStateMutability.Pure
+            ),
+            ">=0.4.13"
+        ]
+    ],
+    /**
+     * @todo Add support for sha3() and suicide() before Solidity 0.5.0
+     * @see https://github.com/ethereum/solidity/releases/tag/v0.5.0
+     * @see https://docs.soliditylang.org/en/latest/050-breaking-changes.html
+     */
+    [
+        "keccak256",
+        [
+            new FunctionType(
+                undefined,
+                [new PointerType(new BytesType(), DataLocation.Memory)],
+                [new FixedBytesType(32)],
+                FunctionVisibility.Default,
+                FunctionStateMutability.Pure
+            ),
+            ">=0.4.13"
+        ]
+    ],
+    [
+        "sha256",
+        [
+            new FunctionType(
+                undefined,
+                [new PointerType(new BytesType(), DataLocation.Memory)],
+                [new FixedBytesType(32)],
+                FunctionVisibility.Default,
+                FunctionStateMutability.Pure
+            ),
+            ">=0.4.13"
+        ]
+    ],
+    [
+        "ripemd160",
+        [
+            new FunctionType(
+                undefined,
+                [new PointerType(new BytesType(), DataLocation.Memory)],
+                [new FixedBytesType(20)],
+                FunctionVisibility.Default,
+                FunctionStateMutability.Pure
+            ),
+            ">=0.4.13"
+        ]
+    ],
+    [
+        "ecrecover",
+        [
+            new FunctionType(
+                undefined,
+                [
+                    new FixedBytesType(32),
+                    new IntType(8, false),
+                    new FixedBytesType(32),
+                    new FixedBytesType(32)
+                ],
+                [new AddressType(false)],
+                FunctionVisibility.Default,
+                FunctionStateMutability.Pure
+            ),
+            ">=0.4.13"
+        ]
+    ]
+    /**
+     * @todo Add support for revert() and require(). Note their signature changes.
+     * @see https://github.com/ethereum/solidity/releases/tag/v0.4.22
+     */
+]);
+
+export const BuiltinAddressMembers = new Map<string, VersionDependentType>([
+    ["balance", [new IntType(256, false), ">=0.4.13"]],
+    [
+        "staticcall",
+        [
+            new FunctionType(
+                undefined,
+                [new PointerType(new BytesType(), DataLocation.Memory)],
+                [new BoolType(), new PointerType(new BytesType(), DataLocation.Memory)],
```

Comment:
The return type as specified here (`(bool, bytes memory)`) only applies after 0.5.0. Before 0.5.0 it was just `bool`. So we will need a way to specify multiple different types, depending on the version.

Reply:
After discussion with @blitz-1306  we decided to fix this as a later PR